setup.py

```diff
@@ -24,45 +23,37 @@ def read(*filenames, **kwargs):
 
 
 root = os.path.dirname(os.path.realpath(__file__))
-version_module = imp.load_source(
-    "version", os.path.join(root, "{{ PACKAGE }}", "version.py"))
-testing = "test" in sys.argv or "pytest" in sys.argv
+version = runpy.run_path(os.path.join(
+    root, 'nengo_bones', 'version.py'))['version']
 
-install_requires = [
-]
-docs_require = [
-    "sphinx",
-    "nengo_sphinx_theme",
-]
-optional_requires = [
-]
-tests_require = [
-    "pytest>=3.6",
-]
+install_requires = ["click>=7.0", "jinja2>=2.10", "pyyaml>=5.1"]
 
+# note: these are intentionally empty to test that the correct requirements
+# are built into the ci scripts
+tests_require = []
+docs_require = []
```

Comment:
This confuses me a bit. If I want to build the docs for nengo-bones locally, how do I make sure I have all the right dependencies? Normally I would do `pip install -e .[docs]` but that does not work here. Instead I would need to locally generate the travis CI script and then run that?

Reply:
Yes for now you would need to manually install the requirements.  I suspect it's going to be pretty rare that anyone wants to build the `nengo-bones` docs locally though, so the importance of checking that we've defined the right requirements in the CI scripts seems worth making the install process a bit more manual for one or two developers.  And as we talked about, this will all go away once we switch to templating `setup.py` as well (since we will no longer need to define requirements in the ci scripts).

Reply:
Ah I completely forgot about the branch `gh-pages-test`. I had been building the docs locally to check my work. Pretty nice feature that I'd like to have in `nengolib`. Are these branches documented anywhere?

Reply:
No, that's probably another documentation section we should add to the nengo-bones docs at some point, a description of all the different features available.